source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,190 @@
+.. _entity-framework-indexes:
+
+=============================
+Optimize Queries with Indexes
+=============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: EF, EF Core, index, optimization, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **indexes** in the {+provider-short+}.
+Indexes can improve the efficiency of queries and add additional functionality
+to querying and storing documents.
+
+Without indexes, MongoDB must scan every document in a collection to find the
+documents that match each query. These collection scans are slow and can
+negatively affect the performance of your application. However, if you create an
+index that covers a query, MongoDB can use the index to limit the
+documents it must inspect.
+
+To improve query performance, build indexes on fields that appear often in your
+application's queries and operations that return sorted results. Each index that
+you add consumes disk space and memory when active, so we recommend that you
+track index memory and disk usage for capacity planning.
+
+Create an Index
+---------------
+
+The {+provider-short+} supports the following types of indexes:
+
+- :ref:`Single field indexes <entity-framework-single-field-index>`
+- :ref:`Compound indexes <entity-framework-compound-index>`
+- :ref:`Unique indexes <entity-framework-unique-index>`
+
+The examples in this guide use the sample application created in the
+:ref:`<entity-framework-quickstart>` guide. After you set up the quick start
+application, you can run the examples in this guide by adding the code to the
+``OnModelCreating()`` method of the ``PlanetDbContext``, as shown in the following
+example:
+
+.. literalinclude:: /includes/fundamentals/code-examples/indexes.cs
+   :language: csharp
+   :start-after: start-model-creating
+   :end-before: end-model-creating
+
+.. note::
+
+   The {+provider-short+} creates your indexes when you call the
+   ``Database.EnsureCreated()`` method. You cannot modify or delete indexes by
```

Comment:
Let's clarify Database.EnsureCreated() is on the dbContext - Database is a property on that rather than a static class.